src/steamship/agents/schema/agent.py

```diff
@@ -49,14 +50,13 @@ def messages_to_prompt_history(messages: List[Block]) -> str:
         as_strings = []
         for block in messages:
             role = block.chat_role
+            # DON'T RETURN AGENT MESSAGES -- THOSE ARE INTERNAL STATUS MESSAGES
```

Comment:
Oh interesting. It's not what I guessed above. Yeah little doc notes would def be helpful then :)